packages/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -37,6 +38,9 @@ export interface AccordionTitleProps
 
   /** Shorthand for the active indicator. */
   indicator?: ShorthandValue
+
+  /** Shorthand for the icon. */
+  icon?: ShorthandValue
```

Comment:
@kuzhelov
@layershifter
@miroslavstastny
@mnajdova
@alinais

do we have any agreement on the `icon` prop for `Accordion`?

Reply:
We should not introduce a new slot there, `Indicator` already covers it:

![image](https://user-images.githubusercontent.com/14183168/53190827-17051b80-3613-11e9-8dab-c01b918d8e5c.png)
https://codesandbox.io/s/n725p7rz54?module=/example.js

Reply:
The Icon is intended to be separate form the indicator. But maybe Adding Icon in general is not right. Maybe making Accordion usable for the control messages was the wrong direction?